backend/src/main/java/org/cryptomator/hub/entities/LegacyAccessToken.java

```diff
@@ -22,6 +23,13 @@
 		INNER JOIN effective_vault_access a ON a.vault_id = t.vault_id AND a.authority_id = d.owner_id
 		WHERE t.vault_id = :vaultId AND d.id = :deviceId AND d.owner_id = :userId
 		""")
+@NamedNativeQuery(name = "LegacyAccessToken.getByDevice", resultClass = LegacyAccessToken.class, query = """
```

Comment:
Why are we using a NativeQuery here?

Reply:
I see that we are already using it for the other method, but the regarding commit (https://github.com/cryptomator/hub/commit/491739a7c3b372886f23b4a9a1608f0d2ca94d9c) does not give any insight about the change reason.

Reply:
Good observation. @SailReal do you recall why we chose to use a native query? I know it was a workaround for some weird bug, but git blame is not very helpful in particular.

Reply:
It is because of https://github.com/cryptomator/hub/issues/222

Reply:
I see, but the questionable joined inheritance is no longer part of these queries, so I guess the issue might be obsolete? I'll check the generated native queries.

Reply:
I did some promising experiments but I believe solving #222 should be a separate PR.